sdk/keyvault/keyvault-keys/test/utils/recorderUtils.ts

```diff
@@ -0,0 +1,22 @@
+import { env } from "@azure/test-utils-recorder";
+import { retry as realRetry } from "./retry";
+
+export const isRecording = env.TEST_MODE === "record";
+export const isPlayingBack = env.TEST_MODE === "playback";
+
+export async function retry<T>(
+  target: () => Promise<T>,
+  delay?: number,
+  timeout?: number,
+  increaseFactor?: number
+): Promise<T> {
+  return realRetry(target, isPlayingBack ? 0 : delay || 10000, timeout || Infinity, increaseFactor);
+}
+
+export function uniqueString(): string {
```

Comment:
Just reminding - 
Go for `recorder.getUniqueName()` and get rid of the `uniqueString()` implememntation.

Reply:
thanks! I did make an issue for this: https://github.com/Azure/azure-sdk-for-js/issues/4872